src/Response/Response.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace TBolier\RethinkQL\Response;
+
+class Response implements ResponseInterface
+{
+    /**
+     * @var int
+     */
+    private $type;
+
+    /**
+     * @var array
+     */
+    private $data;
+
+    /**
+     * @var array
+     */
+    private $backtrace;
+
+    /**
+     * @var array
+     */
+    private $profile;
+
+    /**
+     * @var array
+     */
+    private $note;
+
+    /**
+     * @param int $type
+     * @param array $data
+     * @param array $backtrace
+     * @param array|null $profile
+     * @param array|null $note
+     */
+    public function __construct(int $t = null, array $r = null, array $b = null, array $p = null, array $n = null)
```

Comment:
I'm sure we can give the parameters more meaningful names :)

Reply:
Unfortunately the `ObjectNormalizer` of symfony needs an exact match when reflecting this entity.